reana_commons/version.py

```diff
@@ -14,4 +14,4 @@
 
 from __future__ import absolute_import, print_function
 
-__version__ = "0.6.0.dev20190916"
+__version__ = "0.6.0.dev20191127"
```

Comment:
We don't need to bump the version until we release, with the change you did in the Dockerfile to RWC and RJC it should install the correct versions of everything already, right?

Reply:
it will work locally, but not on Travis, since ``SHARED_VOLUME_PATH`` was moved to ``reana_commons``.